selfdrive/ui/sunnypilot/qt/widgets/controls.h

```diff
@@ -484,7 +487,16 @@ class OptionControlSP : public AbstractControlSP_SELECTOR {
       });
     }
 
-    hlayout->setAlignment(Qt::AlignLeft);
+    innerLayout->setAlignment(Qt::AlignLeft);
+    if (isInlineLayout) {
+      QFrame *container = new QFrame;
+      container->setLayout(innerLayout);
+      container->setStyleSheet("background-color: #393939; border-radius: 20px;");
+      hlayout->addWidget(container);
+     } else {
+       hlayout->addLayout(innerLayout);
+    }
+
```

Comment:
```suggestion
    } else {
      hlayout->addLayout(innerLayout);
    }
```

Remove empty line and fix indendation